src/pages/create_blog.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+import { collection, addDoc } from "@firebase/firestore";
```

Comment:
we have build reusable addDate no need to us this , please use import addDocument from "@/firebase/addData";

Reply:
I just make a 2 functions to (upload Image,Download Image)

Reply:
already edit it ,done